.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: required
 language: go
 go:
-  - '1.12'
+  - "1.12"
```

Comment:
Why did you change single quote for double quote?

Reply:
Linter choice

Reply:
Then, it should be `1.13.x`